CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Upcoming
 
+* Fix message render for non TextEditor panes
```

Comment:
It was actually broken, there was an uncaught error exception, see the bug in the description for more info